modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/RecentApplications.tsx

```diff
@@ -201,6 +202,15 @@ const RecentApplications = () => {
 		</>
 	);
 
+	const callProducts = (selectCard: string) => {
+		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
+			return <ProductCardPersona />;
+		}
+		else {
+			return <ProductCardBusiness />;
+		}
+	};
+
```

Comment:
Please, make the change below.
```suggestion
	const callProducts = (selectCard: string) => {
		if (cardSelected === insuranceCards[0] || selectCard === 'Personal') {
			return <ProductCardPersona />;
		}
		
		return <ProductCardBusiness />;
	};

```